### PR TITLE
upgrade-emitter-version-to-0.23.0

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: packages/autorest.typescript/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/typescript",
-  "version": "6.0.15",
+  "version": "6.0.17",
```
Comment (qiaozha) [inline packages/autorest.typescript/package.json:3]: It's caused by failing to bump the version in an earlier release https://github.com/Azure/autorest.typescript/commit/71273b38e5d262719e8d807c952ae3df925f4d29

**Review by MaryGao — APPROVED** [review]
Approved and only small comment.

File: packages/autorest.typescript/package.json (hunk 1 above)
Comment (MaryGao) [reply packages/autorest.typescript/package.json:3]: I may not get the point, why not 6.0.16?

Comment (qiaozha) [reply packages/autorest.typescript/package.json:3]: In the commit id I comment here, it added the change log but didn't bump the version in package.json.  cause them to be inconsistent, we don't realize this issue in our last release.

**Pull request merged**